website/src/components/ReviewPage/ReviewCard.tsx

```diff
@@ -109,79 +111,59 @@ const ButtonBar: FC<ButtonBarProps> = ({
+    const approvable = sequenceEntryStatus.status === processedStatus && !sequenceEntryStatus.isError;
+    const notProcessed = sequenceEntryStatus.status !== processedStatus;
 
     return (
         <div className='flex space-x-1 mb-auto pt-3.5'>
             <button
-                className={buttonBarClass(sequenceEntryStatus.status !== awaitingApprovalStatus)}
+                className={buttonBarClass(!approvable)}
                 onClick={approveAccessionVersion}
                 data-tooltip-id={'approve-tooltip' + sequenceEntryStatus.accession}
                 key={'approve-button-' + sequenceEntryStatus.accession}
-                disabled={sequenceEntryStatus.status !== awaitingApprovalStatus}
+                disabled={!approvable}
             >
                 <WpfPaperPlane />
             </button>
             <Tooltip
                 id={'approve-tooltip' + sequenceEntryStatus.accession}
                 content={
-                    sequenceEntryStatus.status === awaitingApprovalStatus
+                    approvable
                         ? 'Release this sequence entry'
-                        : sequenceEntryStatus.status === hasErrorsStatus
+                        : sequenceEntryStatus.isError
                           ? 'You need to fix the errors before releasing this sequence entry'
                           : 'Still awaiting preprocessing'
                 }
             />
             {!sequenceEntryStatus.isRevocation && (
                 <button
-                    className={buttonBarClass(
-                        sequenceEntryStatus.status !== hasErrorsStatus &&
-                            sequenceEntryStatus.status !== awaitingApprovalStatus,
-                    )}
+                    className={buttonBarClass(notProcessed)}
                     data-testid={`${getAccessionVersionString({ ...sequenceEntryStatus })}.edit`}
                     data-tooltip-id={'edit-tooltip' + sequenceEntryStatus.accession}
                     key={'edit-button-' + sequenceEntryStatus.accession}
                     onClick={editAccessionVersion}
-                    disabled={
-                        sequenceEntryStatus.status !== hasErrorsStatus &&
-                        sequenceEntryStatus.status !== awaitingApprovalStatus
-                    }
+                    disabled={notProcessed}
                 >
                     <ClarityNoteEditLine />
                 </button>
             )}
             <Tooltip
                 id={'edit-tooltip' + sequenceEntryStatus.accession}
-                content={
-                    sequenceEntryStatus.status !== hasErrorsStatus &&
-                    sequenceEntryStatus.status !== awaitingApprovalStatus
-                        ? 'Cannot edit. Wait for preprocessing!'
-                        : 'Edit this sequence entry'
-                }
+                content={notProcessed ? 'Cannot edit. Wait for preprocessing!' : 'Edit this sequence entry'}
```

Comment:
I did "..." instead of ".." I hope that's fine? 

https://github.com/loculus-project/loculus/pull/3125/commits/2a22284ef44114d8b3c4ef6930a9844fc1beee4c